rosinstalls/indigo/gopher_bootstrap-stable.rosinstall

```diff
@@ -1,29 +1,62 @@
+- git: { local-name: 'rocon_multimaster',     version: 'release/0.9-indigo-gopher',        uri: 'https://github.com/robotics-in-concert/rocon_multimaster.git'}
 
-# Gopher
-- git: { local-name: 'gopher_rapps',          version: 'release/0.2-indigo', uri: 'https://bitbucket.org/yujinrobot/gopher_rapps.git'}
-- git: { local-name: 'gopher_common',         version: 'release/0.5-indigo', uri: 'https://bitbucket.org/yujinrobot/gopher_common.git'}
-- git: { local-name: 'gopher_web_ui',         version: '0.3.0',              uri: 'https://bitbucket.org/yujinrobot/gopher_web_ui.git'}
-- git: { local-name: 'gopher_crazy_hospital', version: 'release/0.2-indigo', uri: 'https://bitbucket.org/yujinrobot/gopher_crazy_hospital.git'}
-- git: { local-name: 'py_trees_suite',        version: 'release/0.3-indigo', uri: 'https://github.com/stonier/py_trees_suite'}
+# Scheduling and Jobs
+- git: { local-name: 'celeros',               version: '0.0.5', uri: 'https://github.com/asmodehn/celeros.git'}
 
-# Rostful and Celeros
-- git: { local-name: 'rostful',               version: '0.0.10',             uri: 'https://github.com/asmodehn/rostful.git'}
-- git: { local-name: 'celeros',               version: '0.0.4',              uri: 'https://github.com/asmodehn/celeros.git'}
-- git: { local-name: 'pyros',                 version: '0.0.9',              uri: 'https://github.com/asmodehn/pyros.git'}
+# Deployment
+- git: { local-name: 'yujin_ansible',     version: '0.1.12',        uri: 'https://bitbucket.org/yujinrobot/yujin_ansible'}
 
 # Environment Integrations
-- git: { local-name: 'elevator_interactions', version: 'release/0.2-indigo', uri: 'https://bitbucket.org/yujinrobot/elevator_interactions.git'}
+# doors is only for documentation builds for now...its along way out of date and we dont use it
+# - git: { local-name: 'door_interactions',     version: 'devel',        uri: 'https://bitbucket.org/yujinrobot/door_interactions.git'}
+- git: { local-name: 'elevator_interactions', version: 'release/0.3-indigo',        uri: 'https://bitbucket.org/yujinrobot/elevator_interactions.git'}
 
-# until fixed version of audio has been released
-- git: { local-name: 'audio_common',          version: 'aab0edb3',           uri: 'https://github.com/yujinrobot/audio_common.git'}
-# until I know why the 1.8.10 deb version doesn't properly unload while the 1.8.10 source version does (debug vs release?)
-- git: { local-name: 'diagnostics',           version: '1.8.10',             uri: 'https://github.com/ros/diagnostics.git'}
+# Until https://github.com/ros-infrastructure/rosdoc_lite/pull/70 is in
+- git: { local-name: 'rosdoc_lite',           version: 'master',       uri: 'https://github.com/yujinrobot/rosdoc_lite.git'}
+# Until we've moved off of indigo (our upstream changes are in, but not in indigo release)
+- git: { local-name: 'audio_common',          version: 'master',       uri: 'https://github.com/yujinrobot/audio_common.git'}
```

Comment:
I'm not sure about this version. It's same as https://github.com/ros-drivers/audio_common. 
There is no related PR and [deb](http://repositories.ros.org/status_page/ros_indigo_default.html?q=audio_common) is released.